wandb/wandb_agent.py

```diff
@@ -540,16 +540,30 @@ def agent(sweep_id, function=None, entity=None, project=None, count=None):
 
     Examples:
         Run a sample sweep over a function:
-        ```
-        def train():
-            with wandb.init() as run:
-                print("config:", dict(run.config))
-                for epoch in range(35):
-                    print("running", epoch)
-                    wandb.log({"metric": run.config.param1, "epoch": epoch})
-                    time.sleep(1)
-
-        wandb.agent(sweep_id, function=train)
+       ```python
```

Comment:
are you missing: `<!--yeadoc-test:...-->`?

Reply:
Yep! This same test appears in two places, so this code as it stands is actually being tested by `one-parameter-sweep.yea`.

But moving forward, these two code snippets might diverge, so I've added `one-parameter-sweep-agent.yea` to track this one.

Reply:
got ya! thanks 👍